app/controllers/cloud_volume_controller.rb

```diff
@@ -19,15 +19,15 @@ def specific_buttons(pressed)
     case pressed
     when 'cloud_volume_attach'
       volume = find_record_with_rbac(CloudVolume, checked_item_id)
-      if !volume.is_available?(:attach_volume) || volume.status != "available"
```

Comment:
@agrare any idea why the UI is checking the volume's status?  isn't that the responsibility of either the AvailabilityMixin or the SupportsFeatureMixin?  Asking because the tests are now broken and I am trying to fix them in a sane way.

Reply:
Yeah I would have expected this to be handled by the `supports :attach_volume` block

Reply:
@agrare any idea which provider this applies to?  I have to assume Openstack, since it is the oldest - but maybe you know.

Reply:
Yes this was openstack, https://github.com/ManageIQ/manageiq-ui-classic/pull/6189